packages/grid/x-data-grid/src/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -175,6 +182,13 @@ const getFilterCallbackFromItem = (
   } else {
     parsedValue = filterItem.value;
   }
+
+  const { ignoreDiacriticsInFiltering } = apiRef.current.getRootProps();
```

Comment:
I have added the `getRootProps` method to be able to access `ignoreDiacriticsInFiltering` here, otherwise, I had to pass the prop through in ~20 places 😅
What do you think?

Reply:
Makes sense. Could have even been just a field like `apiRef.current.rootProps`, this is private anyway.

Reply:
Changed to `apiRef.current.rootProps` instead 👍🏻

Reply:
>  What do you think?

Nice one, it could also be useful in some other places in the future too.